applications/blackhole/src/blackhole_tracking.erl

```diff
@@ -179,8 +179,7 @@ handle_call({'get_sockets', AccountId}, _From, State) ->
     Result =
         case ets:match_object(State, Pattern) of
             [] -> {'error', 'not_found'};
-            Contexts ->
-                [bh_context:websocket_session_id(Context) || Context <- Contexts]
+            Contexts -> Contexts
```

Comment:
Since this returns the `bh_context` objects records, the `get_sockets/1` spec is no longer accurate. Please update it accordingly.